ts/sagas/wallet.ts

```diff
@@ -543,6 +545,25 @@ function* deleteActivePaymentSaga() {
   }
 }
 
+/**
+ * this saga checks the outcome codes coming from a payment or from the payment-check done during the credit card onboarding
+ * if the outcome is different from "success" it tries to delete the payment activation
+ */
+function* shouldDeleteActivePaymentSaga() {
```

Comment:
The `should` prefix reminds me of assertions in tests, it's quite opinionated but what do you think of a name like `deleteUnsuccessfulActivePaymentSaga`? 🙂

Reply:
Definitely better!
6421d8355c22d74edacecf13cf486f588266dc56